src/test/ceph_objectstore_tool.py

```diff
@@ -1796,16 +1797,12 @@ def main(argv):
         vstart(new=False)
         wait_for_health()
 
-        time.sleep(20)
```

Comment:
> if someone notices a regression

this is going to be a painful process since a) the problem is most probably going to be transient, b) the person observing it is not going to figure out this removal could be the cause. 

the removal of these two sleep() is desirable but they deserve a separate pull request, discussion with david and very careful thinking